eng/scripts/Build-Code.ps1

```diff
@@ -3,136 +3,124 @@
 
 [CmdletBinding(DefaultParameterSetName='none')]
 param(
+    [string] $BuildInfoPath,
     [string] $OutputPath,
-    [string] $VersionSuffix,
     [switch] $SelfContained,
     [switch] $SingleFile,
     [switch] $ReadyToRun,
     [switch] $Trimmed,
     [switch] $DebugBuild,
     [switch] $CleanBuild,
-    [switch] $BuildNative,
+    [switch] $Native,
     [string] $ServerName,
-    [Parameter(Mandatory=$true, ParameterSetName='SpecificPlatform')]
+
+    [Parameter(Mandatory, ParameterSetName='SpecificPlatform')]
+    [Alias('OS')]
     [ValidateSet('windows','linux','macOS')]
-    [string] $OperatingSystem,
-    [Parameter(Mandatory=$true, ParameterSetName='SpecificPlatform')]
+    [string[]] $OperatingSystem,
```

Comment:
```suggestion
    [string[]] $OperatingSystems,
```

maybe, since it's an array

Reply:
And, with powershell's parameter prefix matching, callers using "-OperatingSystem" will still work